lib/backend/etcdbk/etcd_test.go

```diff
@@ -384,6 +384,11 @@ func TestCompareAndSwapOversizedValue(t *testing.T) {
 		"dial_timeout":                   500 * time.Millisecond,
 		"etcd_max_client_msg_size_bytes": maxClientMsgSize,
 	})
+	if err != nil && (strings.Contains(err.Error(), "connection refused") || strings.Contains(err.Error(), "context deadline exceeded")) {
```

Comment:
Oh, my bad - did not think to validate them w/o the etcd instance.

On a side note, I think this could benefit if the check was encapsulated into a function (to share the same functionality with the go.check-based setup) and that it would be preferable to make etcd availability an external configuration - e.g. `make test` that spawns the etcd container would set the configuration while the normal `go test` would not (unless 
specified explicitly). Then the configuration could be an environment variable and tests would test for that? WDYT?

Testing for error conditions is till prone to errors as there maybe more cases - i.e. when the etcd is, in fact, running but is having temporary issues - the test would be skipped when it should've failed and the environment inspected.

For example:
```go
func etcdTestEnabled() bool {
	return os.Getenv("TELEPORT_TEST_ETCD") != ""
}
```

Reply:
Yeah, your solution looks better :+1: 
I removed the diff from my PR to avoid conflicts.